src/ape/managers/config.py

```diff
@@ -110,7 +111,11 @@ def _plugin_configs(self) -> Dict[str, ConfigItem]:
             configs[plugin_name] = config
 
         if len(user_config.keys()) > 0:
-            raise ConfigError("Unprocessed config items.")
+            logger.warning(
+                "Not all config items have been processed yet. "
+                "If warning persists after 'ape plugins install', "
+                "there are unrecognized config keys."
```

Comment:
would it make sense to display the keys?

Reply:
Yes, I just improved this a bit